texttospeech/snippets/streaming_tts_quickstart.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# All Rights Reserved.
+
+"""Google Cloud Text-To-Speech API streaming sample application .
+
+Example usage:
+    python streaming_tts_quickstart.py
+"""
+
+
+def run_streaming_tts_quickstart():
+    """Synthesizes speech from a stream of input text.
+    """
+    from google.cloud import texttospeech
+    import itertools
+
+    # Instantiates a client
+    client = texttospeech.TextToSpeechClient()
+
+    # Set the config for your stream. The first request must contain your config, and then each subsequent request must contain text.
+    streaming_config = texttospeech.StreamingSynthesizeConfig(voice=texttospeech.VoiceSelectionParams(name="en-US-Journey-D", language_code="en-US"))
```

Comment:
suggestion: I'm not familiar with this API, so I don't know what valid `name` values are. It might be useful to make sure the `list_voices` example is linked in the documentation above this sample when published.

Reply:
Added comment with link to voices for clarity. FWIW in other samples (eg https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/texttospeech/snippets/long_audio_quickstart.py) we just arbitrarily picked a voice..